checks.d/disk.py

```diff
@@ -176,9 +176,7 @@ def collect_latency_metrics(self):
 
     # no psutil, let's use df
     def collect_metrics_manually(self):
-        df_out = utils.subprocess_output.get_subprocess_output(
-            self.DF_COMMAND + ['-k'], self.log
-        )
+        df_out, err, rtcode = get_subprocess_output(self.DF_COMMAND + ['-k'], self.log)
```

Comment:
Just a general comment when using function that return multiple values: you can discard the values that you don't use with `_`. It's just a convention but makes it clear that you're not using the value afterwards. For instance here you can use:

```python
df_out, _, _ = get_subprocess_output(self.DF_COMMAND + ['-k'], self.log)
```